onnx/defs/tensor/defs.cc

```diff
@@ -3720,4 +3720,65 @@ ONNX_OPERATOR_SET_SCHEMA(
           return true;
         }));
 
+bool BuildContextDependentFunctionBodyAttributeHasValue(
+    const FunctionBodyBuildContext& ctx,
+    const OpSchema& schema,
+    FunctionProto& functionProto) {
+  FunctionBuilder builder(functionProto);
+  auto ints_attr_proto = ctx.getAttribute("ints");
+  bool has_ints = (nullptr != ints_attr_proto) && ints_attr_proto->ints_size() > 0;
```

Comment:
I think we should omit the second check on ints_size?

Reply:
I think I added the check due to test failure. Let me double check.

Reply:
Sorry I misunderstood your comment. I will remove the second check.